ruma-federation-api/src/membership/create_join_event/v1.rs

```diff
@@ -34,7 +34,7 @@ ruma_api! {
     response: {
         /// Full state and auth chain of the room prior to the join event.
         #[ruma_api(body)]
-        #[serde(with = "crate::serde::room_state")]
+        #[serde(with = "crate::serde::v1_pdu")]
         pub room_state: RoomState,
```

Comment:
Hm so is this actually a PDU or do we have to find another name for the serde module?

Reply:
Discussed in the matrix room, we settled on 'PDU'.